paper/paper.md

```diff
@@ -0,0 +1,193 @@
+be calculated. Precision is expressed as the confidence interval around the
+measure. The level of confidence can be specified; the usual 95%-confidence
+interval (CI) is the default. Second, based on a given precision (i.e. CI), the sample size can be calculated. This is mainly of use in the
+planning of prospective studies, where the aim is to estimate a measure of
+interest with enough confidence. The available functions in the package require
+common input arguments; either the sample size or the width of the CI, and the level of the CI. Depending on the function,
+further specific input arguments are required such as the expected area under
+the curve (AUC) for test accuracy.
+
+
+Table 1. Measures, functions and methods provided in presize.
+
+Measure | Function | Methods available 
+-------- | ---------- | --------
+**Descriptive measures** | |
+Mean | `prec_mean` |
+Proportion | `prec_prop` | Wilson, Agresti-Coull, exact, Wald [see @brown2001]
+Rate | `prec_rate` | Score, variance stabilizing, exact, Wald [see @barker2002]
+**Absolute differences** | |
+Mean difference | `prec_meandiff` |
+Risk difference | `prec_riskdiff` | Newcombe [@newcombe1998], Miettinen-Nurminen [@mn1985], Agresti-Caffo [@ac2000], Wald
+**Relative differences** | |
+Odds ratio | `prec_or` | Gart, Wolff, independence smoothed logit [see @fll2015]
+Risk ratio | `prec_riskratio` | Koopman [@koopman1984], Katz [@kbap1978]
+Rate ratio | `prec_rateratio` | Rothman [@rg2018]
+**Correlation measures** | |
+Correlation coefficient | `prec_cor` | Pearson, Kendall, Spearman [see @bw2000]
+Intraclass correlation | `prec_icc` | @bonnett2002
+Limit of agreement | `prec_lim_agree` | @ba1986
+Cohen's kappa | `prec_kappa` | @rd2012
+**Diagnostic measures** | |
+Sensitivity | `prec_sens` | As per `prec_prop`
+Specificity | `prec_spec` | As per `prec_prop`
+Area under the curve | `prec_auc` | @hm1982
+
+
+
+For ease-of-use, we also implemented a Shiny application which can be used
+[online](https://ctu-bern.shinyapps.io/presize) or from within the R-environment.
+Values of parameters can be entered using rulers and numeric fields. Based on
+the given parameters, the application will either display the sample size for a
+given precision, or vice-versa, the precision for a given sample size. Moreover,
+the application also displays the corresponding R-code, which allows copying the
+command into the R-environment for further exploration as well as reproducibility.
+
+
+
+
+# Usage
+
+`presize` is available on [CRAN](https://CRAN.R-project.org/package=presize) or [GitHub](https://github.com/CTU-Bern/presize) and can be loaded into the R session using
+
+```r
+# installation:
+# install.packages("presize") # CRAN
+# remotes::install_github('ctu-bern/presize') # development version on GitHub
+library(presize)
+```
+Suppose we want to estimate the proportion of hospital admissions with diabetes. 
+Diabetes has a prevalence of approximately 10% (@diab) and assume that we have a 
```

Comment:
Consider revising: Diabetes has a prevalence of approximately 10% (@diab). We assume a slightly higher proportion of diabetics...

Reply:
done